packages/kolibri-common/components/SearchBox.vue

```diff
@@ -117,15 +121,29 @@
           textAlign: this.isRtl ? 'right' : '',
         };
       },
+      computedPlaceholder() {
+      if (this.viewportWidth < 1264 && this.viewportWidth >= 840) {
+        return  this.$tr('find');
+      }
+      return this.placeholder || coreString('searchLabel');
+
+    },
     },
     watch: {
       value(current) {
         this.searchInputValue = current || '';
       },
+     viewportWidth(newWidth) {
+      this.viewportWidth = newWidth;
     },
-    created() {
-      this.searchInputValue = this.value;
     },
+  created() {
```

Comment:
After using useKResponsiveWindow, we can remove these listeners and watchers since that composable already take care of that.